cvat/apps/engine/default_settings.py

```diff
@@ -95,3 +95,5 @@
 MAX_CONSENSUS_REPLICAS = int(os.getenv("CVAT_MAX_CONSENSUS_REPLICAS", 11))
 if MAX_CONSENSUS_REPLICAS < 1:
     raise ImproperlyConfigured(f"MAX_CONSENSUS_REPLICAS must be >= 1, got {MAX_CONSENSUS_REPLICAS}")
+
+DEFAULT_DB_BULK_CREATE_BATCH_SIZE = int(os.getenv("CVAT_DEFAULT_DB_BULK_CREATE_BATCH_SIZE", 1000))
```

Comment:
Does this really need to be a setting, and an environment variable? Under what circumstances would one want to adjust this?

Reply:
Basically, it's just a named constant. I can't say why this may require changes so far, so I'm open to removing configurability. Typically, it's better have such performance constants configurable.

Reply:
Alright, I suppose we can keep it.